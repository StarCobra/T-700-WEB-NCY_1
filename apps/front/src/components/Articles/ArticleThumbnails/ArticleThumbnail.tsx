import * as React from 'react';
import '../../../style/article.scss'
import articleExample  from "../../../assets/articles/articleExample.jpg"

export default function articleThumbnail() {
  return (
    <div className="articleThumbnail">
        <img src={ articleExample }></img>
    </div>
  )
}
 