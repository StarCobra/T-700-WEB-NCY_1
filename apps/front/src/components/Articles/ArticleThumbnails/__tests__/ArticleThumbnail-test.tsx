import React from "react";
import { act, create } from "react-test-renderer";
import ArticleThumbnail from "../ArticleThumbnail";

describe("Components > ArticleThumbnails > ArticleThumbnail", () => {
  const tests = [
    {
      description: "generic",
      props: {},
    },
  ];
  tests.forEach((test) => {
    it(`should render the component ${test.description}`, async () => {
      let root;
      await act(async () => {
        root = create(<ArticleThumbnail {...test.props} />);
      });
      expect(root).toMatchSnapshot();
    });
  });
});
