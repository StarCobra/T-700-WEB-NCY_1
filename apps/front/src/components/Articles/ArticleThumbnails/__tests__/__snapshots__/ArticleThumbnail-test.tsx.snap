// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > CryptoArray > CryptoArrayDisplay should render the component generic 1`] = `
<div
  className="articleThumbnail"
>
  <img
    src="default"
  />
</div>
`;
